backports/py-grpcio-testing/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Daiki Maekawa <daikimaekawa29@gmail.com>
+# Maintainer: Daiki Maekawa <daikimaekawa29@gmail.com>
+pkgname=py-grpcio-testing
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="Testing utilities for gRPC Python."
+url="https://grpc.io/"
+arch="all"
+license="Apache License 2.0"
+makedepends="python2-dev py-setuptools cython-dev py2-grpcio py2-protobuf"
```

Comment:
`makedepends` is built time dependencies.
Last two should be `depends`. And version constraint should be specified.

Reply:
Addressed